Lesson Plans/Flight Instructor/CFI/CFI1.4 Teaching Process.md

```diff
@@ -0,0 +1,250 @@
+	   > ✅ Convenient for large groups
+	   > ✅ Can be used to present information that would be difficult for learners to get in other ways (Requires research or restricted reference materials, etc.)
+	   > ✅ Supplementing other teaching methods & devices
+	   > ✅ A brief intro lecture can set the direction and purpose for a demonstration or to start an discussion
+	   > ✅ Efficient to cover many ideas in relatively short time
+	   > ❌ Requires a skilled presenter to maintain attention
+	   > ❌ Difficult to get a precise reading on learners' understanding
+	   > ❌ Does not work well for motor skills
+2. Discussion
+	1. Adding an active participation components to a lesson
+		1. Use a short lecture to start and follow by instructor - learner and learner - learner discussion
+		2. Develops [[ADM|Higher Order Thinking Skills (HOTS)]]
+3. Guided Discussion
+	1. Instead of free-form discussion, instructor can guide and control a discussion session
+	2. Learners would need some level of knowledge to participate and in the process reveals their learning progress
+	3. Planning
+		1. Intro: Gives a brief overview and setup a relaxed, welcoming atmosphere for discussions
+		2. Use lead-off question to open up an area
+		3. Use follow-up questions to guide the direction: dig into learner's explanation or lead the stray discussion back
+		4. Conclusion: Summarize the material covered. Tie together various points and topics discussed.
+
+#### [[Problem-Based Learning]]
+1. [[Scenario-Based Training]]
+2. Collaborative Problem-Solving
+	1. Instructor provides a problem for the group to solve it
+	2. The instructor provides assistance only when needed, but understands that learning to solve a problem without assistance is part of the process
+3. Case-Study
+		1. NTSB aviation accidents is an excellent source of real-world case studies
+
+#### e-Learning
+1. Computer-Assisted Learning (CAL)
+	1. Use computer / tablet with software
+	2. FAA test prep guide software is useful for preparing for knowledge tests
+2. Simulation, Video Games
+	1. Simulators (BATD, AATD, Full-motion Simulator)
+	2. Video Games (X-plane, MSFS, etc.)
+
+#### Cooperative or Group Learning
+1. Organize learners into small groups to maximize understanding
+2. Instructor needs to plan early to determine what group should be able to do on their own
+3. Conditions and controls:
+	1. Small, heterogeneous groups
+	2. Clean, complete instructions: What to do, in what order, with what materials
+	3. Individual accountability
+	4. Recognition and reward for group success
+	5. Time after completion for learners to systematically reflect upon how they worked together as a team
+
+#### Demonstration-Performance
+1. Best used with mental or physical skills that requires practice
+2. Five Phases: Explanation, Demonstration, Learner Performance, Instructor Supervision and Evaluation
+
+#### Drill and Practice
+1. Law of exercise: Connections are strengthened with practice
```

Comment:
Maybe link to laws of learning page? Or/and [[CFI1.3 Learning Process#Types of Practice]] ?